Assets/Scripts/Managers/AdultManager.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class AdultManager : SingletonBehaviour<AdultManager>
+{
+	[SerializeField] GameObject _adultPrefab = null;
+
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float _goodBadCutoff = 0.5f;
+
+	private List<AdultSpawnTag> _spawnPoints = new List<AdultSpawnTag>();
+	private List<ResourceSpawner> _resourceSpawners = new List<ResourceSpawner>();
```

Comment:
I disagree. It doesn't gain us anything and only saves one line later on. I get the desire to save the data we care about, but I don't think it gains us much here.